modules/components/pages/outputs/snowflake_streaming.adoc

```diff
@@ -0,0 +1,500 @@
+
+
+ifndef::env-cloud[]
+Introduced in version 4.39.0.
+endif::[]
+
+[tabs]
+======
+Common::
++
+--
+
+```yml
+# Common configuration fields, showing default values
+output:
+  label: ""
+  snowflake_streaming:
+    account: ORG-ACCOUNT # No default (required)
+    user: "" # No default (required)
+    role: ACCOUNTADMIN # No default (required)
+    database: "" # No default (required)
+    schema: "" # No default (required)
+    table: "" # No default (required)
+    private_key: "" # No default (optional)
+    private_key_file: "" # No default (optional)
+    private_key_pass: "" # No default (optional)
+    mapping: "" # No default (optional)
+    init_statement: | # No default (optional)
+      CREATE TABLE IF NOT EXISTS <table-name> (amount NUMBER);
+    schema_evolution:
+      enabled: false # No default (required)
+      new_column_type_mapping: |-
+        root = match this.value.type() {
+          this == "string" => "STRING"
+          this == "bytes" => "BINARY"
+          this == "number" => "DOUBLE"
+          this == "bool" => "BOOLEAN"
+          this == "timestamp" => "TIMESTAMP"
+          _ => "VARIANT"
+        }
+    batching:
+      count: 0
+      byte_size: 0
+      period: ""
+      check: ""
+    max_in_flight: 64
+```
+
+--
+Advanced::
++
+--
+
+```yml
+# All configuration fields, showing default values
+output:
+  label: ""
+  snowflake_streaming:
+    account: ORG-ACCOUNT # No default (required)
+    user: "" # No default (required)
+    role: ACCOUNTADMIN # No default (required)
+    database: "" # No default (required)
+    schema: "" # No default (required)
+    table: "" # No default (required)
+    private_key: "" # No default (optional)
+    private_key_file: "" # No default (optional)
+    private_key_pass: "" # No default (optional)
+    mapping: "" # No default (optional)
+    init_statement: | # No default (optional)
+      CREATE TABLE IF NOT EXISTS <table-name> (amount NUMBER);
+    schema_evolution:
+      enabled: false # No default (required)
+      new_column_type_mapping: |-
+        root = match this.value.type() {
+          this == "string" => "STRING"
+          this == "bytes" => "BINARY"
+          this == "number" => "DOUBLE"
+          this == "bool" => "BOOLEAN"
+          this == "timestamp" => "TIMESTAMP"
+          _ => "VARIANT"
+        }
+    build_parallelism: 1
+    batching:
+      count: 0
+      byte_size: 0
+      period: ""
+      check: ""
+      processors: [] # No default (optional)
+    max_in_flight: 64
+    channel_prefix: "" # No default (optional)
+```
+
+--
+======
+
+`<table-name>`: Replace the placeholder value with the name of the Snowflake table you want to write data to.
+
+
+== Supported data formats for Snowflake columns
+
+Your output must create messages with the correct data formats for the Snowflake columns you want to write data to.
+
+|===
+| Snowflake column type | Allowed data formats in Redpanda Connect
```

Comment:
For this - do you think it's simpler to reference https://deploy-preview-94--redpanda-connect.netlify.app/redpanda-connect/guides/bloblang/methods/#type here?

Maybe the types we specify here should match up with the bloblang types, because that's essentially what we're doing.

That would mean the table looks like:

| [Snowflake column type](https://docs.snowflake.com/en/sql-reference/intro-summary-data-types) | [Bloblang type](https://deploy-preview-94--redpanda-connect.netlify.app/redpanda-connect/guides/bloblang/methods/#typ) |
|--------|--------|
| CHAR, VARCHAR | string |
| BINARY | string or bytes |
| NUMBER | number or string where string is parsed into a number |
| FLOAT | number |
| BOOLEAN | bool or number where non zero number is `true` | 
| TIME, DATE, TIMESTAMP | timestamp or number where number is converted into a unix timestamp | 
| VARIANT, ARRAY, OBJECT | Any datatype converted into JSON | 
| GEOGRAPHY, GEOMETRY | Not supported |

Reply:
I've updated the table as requested.